cluster/addons/dns/coredns/coredns.yaml.base

```diff
@@ -114,9 +116,18 @@ spec:
           operator: "Exists"
       nodeSelector:
         beta.kubernetes.io/os: linux
+      affinity:
+        podAntiAffinity:
```

Comment:
Has the performance issue (e.g. https://github.com/kubernetes/kubernetes/issues/54164) on larger cluster been resolved yet?
/cc @wojtek-t

Reply:
@ahg-g - for thoughts
There were some improvements, but I don't know if it's good enough already.

Reply:
I've removed the pod anti-affinity changes from this PR.
Once the manual scale tests by @oxddr in https://github.com/kubernetes/kubernetes/pull/84733 are complete, I will include the anti-affinity changes in a separate PR.